File: poseidon/poseidon.cs
using System;

namespace HelloWorld
{
  class Program
  {
    static void Main(string[] args)
    {
        float tempSeidon = 25.475f;
        float tempCelsius = (tempSeidon / 1.37f);
        Console.WriteLine(String.Format("{0:0.##}", tempCelsius));
    }
  }
}

File: poseidon/poseidon.cpp
#include <iostream>
#include <iomanip>

int main()
{
    double tempSeidon = 11.475 + 14.0;
    double tempCelsius = tempSeidon / 1.37;
    std::cout << std::setprecision(2) << std::fixed;
    std::cout << tempCelsius << std::endl;
    return 0;
}
